ruby-challenges.rb

```diff
@@ -10,6 +10,25 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# PsuedoCode:
+# the creation of a method
+#  def a variable that connects the num1,2,and 3 
+# use modulo identify odd or even numbers
+# using if/ elsif to seperate the logic.
+# push p
+
+def number_check(num1, num2, num3)
+  if num % 2 == 0
+    puts "#{num1} is even"
+  elsif num % 2 == 1
+    puts "#{num2} is odd"
+    elsif num % 2 == 2
+        puts "#{num3} is odd"
+  end
+end
```

Comment:
Keep in mind that a Ruby method is going to look and feel very similar to how you would approach a Javascript function, meaning, we want to make it dynamic.  In this problem, you should only have one input passed as an argument, and have only one param that you are checking.